backend/services/allocationService.js

```diff
@@ -266,6 +269,44 @@ const serviceFactory = (elite2Api, keyworkerApi, offenderSearchResultMax) => {
     }
   }
 
+  const searchOffendersPaginated = async (context, { agencyId, keywords, locationPrefix, pageRequest }) => {
```

Comment:
It's rubbish that we don't have any existing tests for these functions (unless I am missing something), but the int tests should cover what we are adding, yes? If not, we ought to add tests.